.github/workflows/main.yml

```diff
@@ -0,0 +1,160 @@
+name: Node CI
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Use Node.js 10.x
+        uses: actions/setup-node@v1
+        with:
+          version: 10.x
+      - name: Yarn install
+        run: yarn
```

Comment:
Where does the `yarn` bin come from?

Reply:
/shrug not sure. I thought it was coming from `actions/setup-node` but I don't see it anywhere in [the repo for that](https://github.com/actions/setup-node)

Reply:
Confirm, setup-node doesn't do it. Turns out that `yarn` is preinstalled on some of the environments but not all (specifically not on the windows environments). 

https://help.github.com/en/articles/software-in-virtual-environments-for-github-actions

Reply:
Yea, I think the docs were missing the part about setting the `runs-on` from the matrix's `os` variable when i first set this up and I wasn't actually running in the different environments :p

Another example of difference in environment that came up after i fixed that was that Mac env doesn't have chrome installed.